[vite-plugin] add validation for `configPath`s (#8572)
cloudflare/workers-sdk: .changeset/sharp-dogs-love.md

---
"@cloudflare/vite-plugin": patch
---

> add validation for `configPath`s
> 
> add some validation for the `configPath`s specified in the plugin's options,
> such validation returns helpful error messages that should clearly indicate
> the issue to users

Reviewer: ```suggestion
Add validation for the `configPath` option in the plugin config that clearly indicates any issues.
```
